website/contributing/release-candidate-patch.md

```diff
@@ -29,13 +29,19 @@ git checkout -b <release-branch>
 git cherry-pick <commit-hash>
 ```
 
-### 2. Test the current changes
+### 2. Bump monorepo packages
+
+- Update packages in the monorepo that were modified by the cherry picks by running `yarn bump-all-updated-packages`. All the package bumps should be on the patch level. Read more about the script and how they work [here](./release-updating-packages).
+
+- CI will take care of releasing the new versions on npm once you push the changes to the `0.XX-stable` branch.
```

Comment:
quick note (already mentioned on the side): remember is that in 0.71 it's 2 separate commands, you need to run the bumping one first, then push it up and wait for the packages to get released, then do the alignment one, while in 0.72 it's one command and you just push it after.

(and well then in 0.70 and lower there's nothing 😅)

Reply:
We won't release an RC for 0.71 and 0.70, so I skipped those steps in this document. However, I added them to the `stable-patch` which we will do for those versions.